### ISSUE TITLE
can't start freeswitch in freebsd jail: Failed to set SCHED_OTHER scheduler (Operation not permitted)

### ISSUE THREAD

**Issue by tbswork1** [signalwire/freeswitch#1892]
**Describe the bug**
Install freeswitch binary package in freebsd jail but can't start it, getting ERROR: Failed to set SCHED_OTHER scheduler (Operation not permitted)

**To Reproduce**
```pkg ins freeswitch
```

**Expected behavior**
it can start itself.

**Package version or git hash**
freeswitch-1.10.3_2

**Trace logs**
none

**backtrace from core file**
none

**Comment by marcinkowalczyk** [comment]
I would check for auditd/selinux settings in your linux server.

**Comment by tbswork1** [comment]
It is FreeBSD.

**Comment by ffdixon** [comment]
Here's how we solved this in Ubuntu when running inside LXC.

```
if systemctl status freeswitch.service | grep -q SETSCHEDULER; then
    sed -i "s/^CPUSchedulingPolicy=rr/#CPUSchedulingPolicy=rr/g" /lib/systemd/system/freeswitch.service
    systemctl daemon-reload
  fi
```

Basically, FreeSWITCH uses systemd on Ubuntu and we needed to disable the round robin CPU scheduling parameer `CPUSchedulingPolicy=rr`.

**Comment by WellaRo** [comment]
I'm running FreeSWITCH in FreeBSD since long time ago, and this error message always existed, but it didn't keep FS from running. I believe that the reason for FS to not start is other than the schedule message.

Run in foreground from command line and check messages.
Verify that "/usr/local/etc/freeswitch/current" folder contains the configuration files.

**Comment by tbswork1** [comment]
@WellaRo so you still running FS on FreeBSD? 

We ditched FreeBSD. Run it on Debian now. Running a mission critical system on a platform that lacks support or interest is dangerous and looking for troubles ...

**Comment by WellaRo** [comment]
Yes we do, because is quite useful to have multiple instances of FreeSWITCH inside of Jails, PF firewall, snapshots, etc...  But I totally understand that you can't give support to any other O.S. out there.
